packages/wrangler/src/type-generation/index.ts

```diff
@@ -468,6 +487,7 @@ function writeDTSFile({
 				combinedTypeStrings,
 			].join("\n")
 		);
+		logger.log(`Generating project types...\n`);
```

Comment:
I referred to this as "project types", but open to suggestions of course.

Reply:
Should it match the --x-with-runtime arg and say "Generating runtime types..."

Reply:
This is not for the runtime types, but for the `Env` types, so the wording is for distinguishing between the two.